geoengine/workflow_builder/operators.py

```diff
@@ -31,6 +31,16 @@ def to_workflow_dict(self) -> Dict[str, Any]:
             'operator': self.to_dict(),
         }
 
+    @classmethod
+    def from_workflow_dict(cls, workflow) -> 'Operator':
```

Comment:
```suggestion
    def from_workflow_dict(cls, workflow) -> Operator:
```

from SO

> The third solution is Python 3.7+ only -- use the from __future__ import annotations directive:

```Python
from __future__ import annotations 

class Foo:
    def bar(self) -> Foo:
        return Foo()
```

> This will automatically make all annotations be represented as strings. So we get the benefit of the first solution, but without the ugliness.